### DISCUSSION TITLE
k3s.yaml RBAC

### DISCUSSION THREAD

**Discussion by cwrx777** [k3s-io/k3s#10445] [Q&A]
Hi,

what kind of kube API access when we use the k3s.yaml created during server installation?
I notice there is a user named `default` in the file. is it mapped to any one of of subject in ClusterRoleBindings or ServiceAccounts ?

**Comment by brandond** [comment]
The `user` in the kubeconfig struct doesn't matter; the subject on the client certificate is all that matters to the apiserver. The admin kubeconfig uses the special `system:admin` user and `system:masters` group which is treated specially by the apiserver, and bypassess all RBAC checks.
